index.html

```diff
@@ -52,5 +58,213 @@ <h2 class="typography supportText">I’am a Software Developer</h2>
+				</div>
+				<div class="btn-courses ">
+					<button type="button" class="function">HTML</button>
+					<button type="button" class="function">bootstrap</button>
+					<button type="button"class="function">Ruby</button>
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+			</div>
+		  
+			<div class="post">
+				<div class="post-title">
+					<h1>Professional Printing Art</h1>
+					<h1>Printing Data</h1>
+				</div>
+				<div class="post-content">
+					<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				</div>
+				<div class="btn-courses">
+					<button type="button" class="function">HTML</button>
+					<button type="button" class="function">bootstrap</button>
+					<button type="button" class="function">Ruby</button>
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+			</div>
+			
+			<div class="post">
+				<div class="post-title">
+					<h1>Professional Printing Art</h1>
+					<h1>Printing Data</h1>
+				</div>
+				<div class="post-content">
+					<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				</div>
+				<div class="btn-courses">
+					<button type="button"class="function">HTML</button>
+					<button type="button"class="function">bootstrap</button>
+					<button type="button"class="function">Ruby</button>
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+			</div>
+		
+			<div class="post">
+				<div class="post-title">
+					<h1>Professional Printing Art</h1>
+					<h1>Printing Data</h1>
+				</div>
+				<div class="post-content">
+					<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				</div>
+				<div class="btn-courses">
+					<button type="button" class="function">HTML</button>
+					<button type="button" class="function">bootstrap</button>
+					<button type="button" class="function">Ruby</button>
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+			</div>
+			<div class="post">
+				<div class="post-title">
+					<h1>Proffesional Printing Art</h1>
+					<h1>Printing Data</h1>
+				</div>
+				<div class="post-content">
+					<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				</div>
+				<div class="btn-courses">
+					<button type="button" class="function">HTML</button>
+					<button type="button" class="function">bootstrap</button>
+					<button type="button" class="function">Ruby</button>
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+				</div>
```

Comment:
- [ ] [**OPTIONAL**] Good job setting up this project :clap:, but I noticed that the images you provided broken links. Kindly update that to make your portfolio more proffessional :+1: 

![Screenshot from 2023-01-18 18-09-08](https://user-images.githubusercontent.com/80458199/213243152-c7125768-a71a-41b0-93f5-abeebf519f7f.png)

Reply:
hey iambenkis,
I saw it too but it was a small problem because I didn't merge the images branch to the main branch. but I merged it now its working well pictures can be seen.
Thank you!
cheers